src/api/data-migration/version-1-to-version-2/create-new-application-relationships/export-contract-agent/index.ts

```diff
@@ -0,0 +1,64 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import createInitialAgents from './create-initial-agents';
+import createAgentServices from './create-agent-services';
+import createAgentServiceCharges from './create-agent-service-charges';
+import { Application } from '../../../../types';
+
+/**
+ * createExportContractAgent
+ * Create a new "export contract agent" with "service" and "service charge" relationships.
+ * 1) Create an array of export contract ID "connect" relationships.
+ * 2) Create initial agent entries.
+ * 3) Create an array of agent ID "connect" relationships.
+ * 4) Create agent service entries.
+ * 5) Create an array of service ID "connect" relationships.
+ * 6) Create agent service charge entries.
+ * @param {Connection} connection: SQL database connection
+ * @param {Array<Application>} applications: Applications
+ * @returns {Promise<Array<ApplicationExportContractAgent>>} Export contract agent entries
+ */
+const createExportContractAgent = async (context: Context, applications: Array<Application>) => {
+  const loggingMessage = 'Creating exportContractAgent with service and charge relationships';
+
+  console.info(`✅ ${loggingMessage}`);
```

Comment:
just wondering, is it worth having a tick console.info at the bottom before the return so we know which file has started and then completed etc

Reply:
Had a look into this, tried adding this to e.g `connectToDatabase`. The `process.nextTick()` is logged _after_ other logs we have ("✅ Connected to database").

There are calls to the database before `nextTick` that would fail if not connected to the database. It seems it wouldn't work as expected, unless i'm missing something